.github/actions/create-ingest-cache/action.yml

```diff
@@ -0,0 +1,46 @@
+
+name: 'Ingest Cache Build'
+description: 'Build ingest specific cache on top of base cache'
+inputs:
+  python-version:
+    description: 'which python version to user'
+    required: true
+  force-recreate:
+    description: 'regardless of cache existing, recreate it'
+    default: "false"
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/cache/restore@v3
+      id: virtualenv-cache
+      with:
+        path: |
+          nltk_data
+          .venv
+        key: unstructured-${{ runner.os }}-${{ inputs.python-version }}-${{ env.GHA_CACHE_KEY_VERSION }}-${{ hashFiles('requirements/*.txt') }}
+    - name: Setup virtual environment (no cache hit)
+      if: steps.virtualenv-cache.outputs.cache-hit != 'true'
+      uses: ./.github/actions/create-cache
+      with:
+        python-version: ${{ inputs.python-version }}
+        force-recreate: 'true'
+    - uses: actions/cache@v3
+      id: ingest-virtualenv-cache
+      with:
+        path: |
+          .venv
+          nltk_data
```

Comment:
realize this would have been true before, but does the ingest-specific cache end up with only the additional ingest-specific content? or is the ingest cache actually the base cache + the ingest content?